# Goal
string target_map
float64 x
float64 y
float64 yaw
---
# Result
bool success
string message
---
# Feedback
string status